Dockerfile/files/startup.sh

```diff
@@ -1,18 +1,12 @@
 #!/bin/bash
 
-if [[ ! -d /botsrc/.pyenv ]]; then
-    git clone https://github.com/pyenv/pyenv.git /botsrc/.pyenv
-fi
-
-echo 'export PYENV_ROOT="/botsrc/.pyenv"' >> ~/.bashrc
-echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bashrc
-echo 'eval "$(pyenv init -)"' >> ~/.bashrc
 source ~/.bashrc
 
-pyenv install -s 3.7.1
 pyenv shell 3.7.1
 
 cd /botsrc/
 pip3 install -r requirements.txt
 
+cp -r /cache/.pyenv /botsrc/
```

Comment:
Copying the pyenv folder at each container startup is not very efficient. Another disadvantage is that if requirements.txt changes, you would need to build a new container.

If there are problems with this mechanism, perhaps there are other methods of avoiding them?